compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/ir/IrUtils.kt

```diff
@@ -484,4 +503,4 @@ fun IrClass.addFakeOverrides() {
 }
 
 fun IrValueParameter.isInlineParameter() =
```

Comment:
@sfs As I see this method is only used in JVM backend so it's best to move it back to JVM utils cause extension inlining could depend on backend: ```index >=0```

Reply:
That's a good point, I've moved the function to the IrUtils file in the jvm backend.